ttnn/cpp/ttnn/operations/matmul/matmul_pybind.cpp

```diff
@@ -158,8 +158,8 @@ void py_module(py::module& module) {
         If the input tensors have more than two dimensions, the additional, front,
         dimensions may be used for batched matrix multiply.
         These front dimensions may also be referred to as batch dimensions.
-        E.g. a tensor with dimensions :math:`(a \\times b \\times c \\times d)`
-        has batch dimensions a and b.
+        E.g. a tensor with dimensions (`a` x `b` x `c` x `d`)
+        has batch dimensions `a` and `b`.
         The following are the allowed possibilities for batch dimensions.
         Examples below show concrete operations and tensor sizes.
 
```

Comment:
For line 166, please change "If all batch dimensions are all of size" to "If all batch dimensions are of size"